startly_style.css

```diff
@@ -0,0 +1,218 @@
+@import url('http://fonts.googleapis.com/css?family=Economica');
+@import url('http://fonts.googleapis.com/css?family=Monoton');
+@import url('http://fonts.googleapis.com/css?family=Tangerine');
+
+
+* {
+  font-family: Economica;
+  background-color: seashell;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.topper {
+  display: grid;
+  grid-template-columns: 1fr 2fr 1fr;
+  padding-left: 0.5em;
+  background-color: LavenderBlush;
+  /* position:fixed; */
+}
+
+
+ul li {
+  list-style-type: none;
+  margin: 0;
+  padding: 1.0em;
+  font-size: 1.2rem;
+}
+
+#try {
+  grid-column-start: 2;
+  text-align: center;
+}
+
+#about {
+  grid-column-start: 3;
+  text-align: right;
+}
+
+.banner {
+  display: flex;
+  flex-direction: column;
+  background-color: LavenderBlush;
+  align-items: center;
+  padding-bottom: 0.4em;
+  padding-top: 0.4em;
+  border-bottom: 1px solid gray;
+}
+
+.banner sub {
+  line-height: 2;
+  font-size: 1.2rem;
+  padding-bottom: 0.8em;
+  background-color: LavenderBlush;
+}
+
+.banner h1 {
```

Comment:
There might be a way to put background-color: LavenderBlush on the parent element so that you can DRY up the code.